Maker-Agents-Suit-Module/OSMAgent/src/rely.function.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  Finding,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  HandleTransaction,
+} from "forta-agent";
+
+import {
+  provideFunctionCallsDetectorAgent,
+} from "@nethermindeth/general-agents-module";
+
+export const RELY_FUNCTION_SIG = "rely(address)";
+
+export const createFinding = (
+  metadata: { [key: string]: any } | undefined
+): Finding => {
+  return Finding.fromObject({
+    name: "Maker OSM Contract RELY Function",
+    description: "RELY Function is called",
+    alertId: "MakerDAO-OSM-3",
+    severity: FindingSeverity.Medium,
+    type: FindingType.Unknown,
+    metadata: {
+      contract: metadata ? metadata.to : null,
+    },
+  });
```

Comment:
`everestId` is missing
```suggestion
  return Finding.fromObject({
    name: "Maker OSM Contract RELY Function",
    description: "RELY Function is called",
    alertId: "MakerDAO-OSM-3",
    severity: FindingSeverity.Medium,
    type: FindingType.Unknown,
    everestId: "0xbabb5eed78212ab2db6705e6dfd53e7e5eaca437",
    metadata: {
      contract: metadata ? metadata.to : null,
    },
  });
```